mods/d2k/rules/vehicles.yaml

```diff
@@ -337,6 +337,11 @@ devastator:
 		Step: 5
 		Ticks: 3
 		HealIfBelow: 50%
+	AttackDetonates:
```

Comment:
Detonation is a manually issued deploy order, so it doesn't need/want to be an attack trait at all.